src/site/content/en/blog/contact-picker/index.md

```diff
@@ -0,0 +1,316 @@
+---
+title: A contact picker for the web
+subhead: The Contact Picker API is an on-demand API that allows users to select entries from their contact list and share limited details of the selected entries with a website. It allows users to share only what they want, when they want, and makes it easier for users to reach and connect with their friends and family.
+authors:
+  - petelepage
+description: Access to the user's contacts has been a feature of native apps since (almost) the dawn of time. The Contact Picker API is an on-demand API  that allows users to select an entry or entries from their contact list and share limited details of the selected contact(s) with a website. It allows users to share only what they want, when they want, and makes it easier for users to reach and connect with their friends and family.
+date: 2019-08-07
+updated: 2019-10-08
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - capabilities
+  - contacts
+  - chrome77
+hero: hero.jpg
+alt: Telephone on yellow background.
+---
+
+
+{% Aside %}
+  The Contact Picker API begins an origin trial in Chrome 77
+  (stable in September) as part of our capabilities project. We'll keep this
+  post updated as the implementation progresses.
+{% endAside %}
+
+## What is the Contact Picker API? {: #what }
+
+<style>
+  #video-demo { max-height: 600px; }
+  figcaption { font-size: small; }
+</style>
+
+<a href="https://storage.googleapis.com/webfundamentals-assets/contact-picker/contact-picker.mp4">
+  <video id="video-demo" loop autoplay muted class="w-figure w-figure--inline-right" style="max-width:50%"
```

Comment:
This video is a little tricky on mobile because it's so vertical.

![image](https://user-images.githubusercontent.com/1066253/66456903-cd433700-ea23-11e9-814c-6006a0571afa.png)

One idea would be to hide it on smaller viewports?

Reply:
@robdodson regarding "hiding it on smaller viewports" is that site code that you will add or is this something that Pete should do? How should he do that?

Also, just brainstorming other ideas, rather than hiding on mobile maybe we can just link out to the video?

Reply:
Added a comment above to offer another suggestion on how to properly center the video. I suspect if we do that then it'll look ok.

Reply:
I think we've got this fixed now